addons/bus/controllers/main.py

```diff
@@ -38,5 +38,5 @@ def poll(self, channels, last, options=None):
         if [c for c in channels if not isinstance(c, pycompat.string_types)]:
             raise Exception("bus.Bus only string channels are allowed.")
         if request.registry.in_test_mode():
-            raise exceptions.UserError("bus.Bus not available in test mode")
+            raise exceptions.UserError(_("bus.Bus not available in test mode"))
```

Comment:
Is there really a point in translating this? If you think so, perhaps it would be nice to change the text to something user-friendly, before making translators work on it ;-)

Reply:
too late. It's already live in master

Reply:
@qdp-odoo it's never too late :stuck_out_tongue: